cmds/wifi/wifi.go

```diff
@@ -6,18 +6,28 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"os/exec"
 )
 
-const cmd = "wifi [options] essid passphrase"
+const (
+	cmd          = "wifi [options] essid [passphrase]"
+	nopassphrase = `network={
+	ssid="%s"
+	proto=RSN
+	key_mgmt=NONE
+}
+`
+)
 
 var (
 	iface = flag.String("i", "wlan0", "interface to use")
 	essid string
 	pass  string
+	conf  []byte
```

Comment:
all three of these could just be local variables in main?

Reply:
other than that, looks all good.

Reply:
great suggestion, done.